source/crud/tests/unified/bulkWrite-update-hint-clientError.yml

```diff
@@ -0,0 +1,151 @@
+# This file was created automatically using mongodb-spec-converter.
+# Please review the generated file, then remove this notice.
+
+description: bulkWrite-update-hint-clientError
+schemaVersion: '1.1'
+createEntities:
+  -
+    client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      uriOptions:
+        retryReads: false
+      observeEvents:
+        - commandStartedEvent
+  -
+    database:
+      id: &database0 database0
+      client: client0
+      databaseName: &database_name crud-v2
+  -
+    collection:
+      id: &collection0 collection0
+      database: database0
+      collectionName: &collection_name test_bulkwrite_update_hint
+runOnRequirements:
+  -
+    maxServerVersion: 3.3.99
+initialData:
+  -
+    collectionName: *collection_name
+    databaseName: *database_name
+    documents:
+      -
+        _id: 1
+        x: 11
+      -
+        _id: 2
+        x: 22
+      -
+        _id: 3
+        x: 33
+      -
+        _id: 4
+        x: 44
+tests:
+  -
+    description: 'BulkWrite updateOne with update hints unsupported (client-side error)'
+    operations:
+      -
+        object: *collection0
+        name: bulkWrite
+        arguments:
+          requests:
+            -
+              updateOne:
+                filter: &updateOne_filter
+                  _id: 1
+                update: &updateOne_update
+                  $inc:
+                    x: 1
+                hint: &hint_string _id_
+            -
+              updateOne:
+                filter: *updateOne_filter
+                update: *updateOne_update
+                hint: &hint_doc
+                  _id: 1
+          ordered: true
+        expectError:
+          isError: true
```

Comment:
In contrast to my previous comment about server-side errors, this would be a case to use `isClientError: true` instead.